src/hw1/bash_commands.py

```diff
@@ -0,0 +1,35 @@
+def head(file, strings=10):
+    for i in range(strings):
+        print(file.readline(), end="")
+
+
+def tail(file, strings=10):
+    lines = [line for line in file]
+    lines.reverse()
+    for i in range(strings):
+        print(lines[i], end="")
+
+
+def wc(file):
+    wordCount = 0
+    symbolCount = 0
+    lines = [line for line in file]
```

Comment:
This makes you save all lines in one place. What if the file is 50GB in size? It is better to do this iteratively.